examples/analog_out/gen_voltage_wfm_int_clk.py

```diff
@@ -0,0 +1,17 @@
+"""Example of analog output voltage generation.
+
+This example demonstrates how to output a finite number of
+voltage samples to an Analog Output Channel using an internal
+sample clock.
+"""
+
+import nidaqmx
+from nidaqmx.constants import AcquisitionType
+
+with nidaqmx.Task() as task:
+    task.ao_channels.add_ao_voltage_chan("Dev1/ao0")
+    task.timing.cfg_samp_clk_timing(1000, sample_mode=AcquisitionType.FINITE, samps_per_chan=1000)
+
+    print(f"Generate {task.write([1.1, 2.2, 3.3, 4.4, 5.5])} voltage samples.")
```

Comment:
not a big fan of an f-string having such a massive side-effect. can you make this two statements? Also, I would be explicit about auto-start optional variable.

```suggestion
    number_of_samples_written = task.write([1.1, 2.2, 3.3, 4.4, 5.5], auto_start=True)
    print(f"Generate {number_of_samples_written} voltage samples.")
```

Reply:
I agree. Printing to the console is presentation logic and removing it should not affect the business logic. Please remove all calls from `task.write` from f-strings.

Reply:
Sure, it's updated as per suggestion.